pkg/timeutil/timeutil.go

```diff
@@ -20,9 +24,32 @@ func Date(datetime time.Time) time.Time {
 func LastWeekday(weekday time.Weekday) time.Time {
 	date := Date(time.Now())
 
-	for date.Weekday() != weekday {
+	for date.Weekday() !=  weekday{
 		date = date.AddDate(0, 0, -1)
 	}
 
 	return date
 }
+
+// StringToWeekday transforms a string to time.Weekday
+// it returns Monday if wrong day specified in config.toml
+func StringToWeekday(weekday string) (time.Weekday, error){
```

Comment:
another suggestion here is to create a `map[string]time.Weekday` in a variable in the package and lookup in that map, which is similar to the `switch` but will perform better :-)